notebooks/noj_book/sklearn_reference.clj

```diff
@@ -10,11 +10,17 @@
    [noj-book.utils.render-tools-sklearn]
    [scicloj.sklearn-clj.ml]))
 
+;;## Sklearn model reference
 
+;;Below we find all sklearn models with their parameters and the original documentation.
```

Comment:
We need more context here:
* Link to the `ml_basic` chapter for background.
* Explain what sklearn is, link to the sklearn docs, and remind the reader that Noj contains `sklearn-clj`, which is a Clojure wrapper of Sklearn.